packages/react/src/components/TextInput/PasswordInput.tsx

```diff
@@ -309,6 +311,24 @@ const PasswordInput = React.forwardRef(function PasswordInput(
       [`${prefix}--tooltip--align-${tooltipAlignment}`]: tooltipAlignment,
     }
   );
+
+  let align: PopoverAlignment | undefined = undefined;
+
+  if (tooltipPosition === 'top' || tooltipPosition === 'bottom') {
+    if (tooltipAlignment === 'center') {
+      align = tooltipPosition;
+    }
+    if (tooltipAlignment === 'end') {
+      align = `${tooltipPosition}-right`;
+    }
+    if (tooltipAlignment === 'start') {
+      align = `${tooltipPosition}-left`;
+    }
+  }
+
+  if (tooltipPosition === 'right' || tooltipPosition === 'left') {
+    align = tooltipPosition;
+  }
```

Comment:
Is this necessary? Tooltip now supports new alignment options like `top-end` and `top-start`.

This code pattern is something we'll eventually want to remove from the entire codebase, I added it as a line item within 
* https://github.com/carbon-design-system/carbon/issues/16009

Reply:
Oh, the prototypes defined in the component were still using the old values. Do you want me to cover these changes as part of this PR or take it as a separate PR with #16009.